docs/swagger.yml

```diff
@@ -931,6 +931,15 @@ definitions:
         type: "integer"
         format: "int64"
         description: "The maximum concurrent host scans. default is 50."
+      disable_optional_products:
+        type: "array"
```

Comment:
I don't think this is correct based on the values below.  Looks like this should be its own defined object as its not an array list due to design changes.